client/payment-details/dispute-details/style.scss

```diff
@@ -48,3 +48,41 @@
 		margin-bottom: 8px;
 	}
 }
+.dispute-steps {
+	&__header {
+		font-weight: 600;
+	}
+	&__steps {
+		list-style-position: inside;
+		margin-left: 0;
+
+		.wcpay-tooltip__content-wrapper.wcpay-tooltip--click__content-wrapper
+			> .wcpay-tooltip__content-wrapper
+			> [role='button'] {
+			margin-left: 0;
+			margin-right: 0;
+			vertical-align: text-bottom;
+		}
+
+		.wcpay-tooltip__content-wrapper.wcpay-tooltip--click__content-wrapper
+			> .wcpay-tooltip__content-wrapper
+			+ div {
+			display: inline;
```

Comment:
This is necessary to fix the line break problem shown here:

https://github.com/Automattic/woocommerce-payments/assets/73803630/8201edfe-ca4e-4c77-a37d-dc9312a07a8c

Reply:
Nice catch! I've moved this to the tooltip component styles in 0b768b3